client/app/dedicatedCloud/security/kms/add/dedicatedCloud-security-kms-add.html

```diff
@@ -67,16 +67,16 @@
                         'currentOperation': $ctrl.kmsCreationTask.name,
                         'state': $ctrl.kmsCreationTask.state
                     }"></strong>
-            <oui-progress class="mb-4 d-block text-nowrap text-truncate" aria-describedby="kmsCreationProgress">
+            <oui-progress class="mb-4 d-block text-nowrap text-truncate">
                 <oui-progress-bar data-type="info"
                                   data-value="$ctrl.kmsCreationTask.progress">
                     <span data-ng-bind="$ctrl.getTaskDetails()"></span>
                 </oui-progress-bar>
             </oui-progress>
-            <span class="d-block text-nowrap text-truncate"
-                  data-ng-if="$ctrl.error && $ctrl.error.data"
-                  data-translate="dedicatedCloud_vm_encryption_kms_add_error"
-                  data-translate-values="{ 'error' : $ctrl.error.data.message }"></span>
+            <p class="d-block text-nowrap text-truncate"
```

Comment:
-`.d-block`

Reply:
Also not a big fan of `.text-truncate`

Reply:
In fact, i forgot to use `oui-paragraph` that can fit my need